### DISCUSSION TITLE
Add a break line in the column Title

### DISCUSSION THREAD

**Discussion by BenOussama180** [rappasoft/laravel-livewire-tables#1157] [Q&A]
so i have a long column title 
![image](https://user-images.githubusercontent.com/81111919/231391257-e53da2ab-9b4b-4cef-a131-59ed8dc46770.png)
i  want to insert break lines to make space in the table , 
i tried this  but not working sadly : 
```php
Column::make("Organic\n Search\n Traffic", "organic_search_traffic")
                ->sortable()
                 .....
```
 @lrljoe any idea on a workaround for this , thanks in advanced!

**Comment by lrljoe** [comment]
Yep, use the 
```php
setThAttributes()
```
Method in the configure() section, and pass "default => false" into the array.

e.g.
```php
$this->setThAttributes(function(Column $column) {
                if ($column->isField('organic_search_traffic')) {
                  return [
                    'class' => 'px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider dark:bg-gray-800 dark:text-gray-400',
                    'default' => false
                  ];
                }
                return [];
              });
```
The bits in the "class" is all of the **default** items, but without "whitespace-nowrap", so it'll allow it to wrap.  Make sure to reference by **FIELD** not Column name here, and keep in mind that the default classes _may_ change

You could also just add "whitespace-break-spaces", but your mileage may vary with some browsers, as some will take override later instances.
```php
$this->setThAttributes(function(Column $column) {
                if ($column->isField('organic_search_traffic')) {
                  return [
                    'class' => 'whitespace-break-spaces',
                  ];
                }
                return [];
              });
```
